### PR TITLE
Fix docker file and init checks

### LINKED ISSUE #141: fix dockerfile
## Description

The Dockerfile should run `make install` to download and compile the RAVEN hydrological model and OSTRICH. This creates binary executables that are essential to the services. 

Is it possible to run the test suite when the docker image is built and report errors ?

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes #141 

Changes:

- dockerfile make command
- __init__ checks

### CODE REVIEW

**Comment by huard** [comment]
Dockerhub built the image !

File: docker-compose.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 version: '3'
 services:
-  emu:
+  raven:
     build: .
-    image: huard/raven
+    image: pavics/raven
```
Comment (tlvu) [inline docker-compose.yml]: I would rather call this image `local_raven`.  Doing it the current way will overlay our local build with the real `pavics/raven:latest` from official Docker Hub build.

**Comment by tlvu** [comment]
Yaahoo, `Time series analysis.ipynb` passed with latest raven docker `0.6.3` and new build of jupyter env to include latest birdy.

**Comment by tlvu** [comment]
`Delineation_workflow.ipynb` failed with Self-signed SSL issue again, is this another fix in birdy?

```
Traceback (most recent call last):
  File "/opt/conda/envs/wps/lib/python3.6/site-packages/pywps/app/Process.py", line 200, in _run_process
    self.handler(wps_request, wps_response)  # the user must update the wps_response.
  File "/opt/wps/raven/processes/wps_generic_shape_properties.py", line 53, in _handler
    shape_url = request.inputs['shape'][0].file
  File "/opt/conda/envs/wps/lib/python3.6/site-packages/pywps/inout/basic.py", line 435, in file
    raise NoApplicableCode('File reference error: %s' % e)
pywps.exceptions.NoApplicableCode: 400 NoApplicableCode: File reference error: HTTPSConnectionPool(host='lvu.ouranos.ca', port=443): Max retries exceeded with url: /wpsoutputs/8ce367c6-8b95-11e9-915c-0242ac190009/input.geojson (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",),))
ERROR: Process._run_process(): Process error: wps_generic_shape_properties.py._handler Line 53 400 NoApplicableCode: File reference error: HTTPSConnectionPool(host='lvu.ouranos.ca', port=443): Max retries exceeded with url: /wpsoutputs/8ce367c6-8b95-11e9-915c-0242ac190009/input.geojson (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",),))
```

**Comment by tlvu** [comment]
`computing_objective_functions.ipynb` fails with similar SSL verify error:

```
ERROR: Process._run_process(): Process error: wps_objective_functions.py._handler Line 54 400 NoApplicableCode: File reference error: HTTPSConnectionPool(host='lvu.ouranos.ca', port=443): Max retries exceeded with url: /wpsoutputs/241a5b5a-8b9a-11e9-a459-0242ac190009/raven-gr4j-cemaneige-sim-0_Hydrographs.nc (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",),))

```

**Comment by tlvu** [comment]
`multi_model_simulation.ipynb`, birdy issues:

```
/usr/local/envs/birdy/lib/python3.7/site-packages/birdy/client/converters.py in convert(self)
    156         try:
    157             # try OpenDAP url
--> 158             return xr.open_dataset(self.url)
    159         except IOError:
    160             # download the file

AttributeError: 'XarrayConverter' object has no attribute 'url'
```

**Comment by tlvu** [comment]
`running_gr4jcn.ipynb` passed !!!

**Comment by huard** [comment]
Will only be able to look into this tomorrow PM.

**Comment by tlvu** [comment]
Merged this PR, the Dockerfile looks good now.  Other fixes seems to be from birdy (SSL, AttributeError: 'XarrayConverter'), twitcher (large POST), raven (different looking graph) so not related to Dockerfile anymore.

**Pull request merged**